widgets/button/button.php

```diff
@@ -243,12 +243,12 @@ function get_less_variables($instance){
 		$less_vars = array(
 			'button_width' => isset( $instance['design']['width'] ) ? $instance['design']['width'] : '',
 			'has_button_width' => empty( $instance['design']['width'] ) ? 'false' : 'true',
-			'button_color' => $instance['design']['button_color'],
-			'text_color' => $instance['design']['text_color'],
+			'button_color' => isset($instance['design']['button_color']) ? $instance['design']['button_color'] : '',
+			'text_color' =>   isset($instance['design']['text_color']) ? $instance['design']['button_color'] : '',
```

Comment:
This should be <pre>'text_color' =>   isset($instance['design']['text_color']) ? $instance['design']['<b>text_color</b>'] : '',</pre>